### ISSUE TITLE
Hardware Info/Overview GUI Panel

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6317]
**The problem you're addressing (if any)**
As a non-technical user, I need to see info about the hardware on my QubesOS device. 

**Describe the solution you'd like**
Anything easily discoverable from the App Menu or Qubes Manager, and elegantly integrated into other Qubes OS level GUI tools.

Screenshots below show nice "overview vs snapshot" presentation of info available in my MacOS equivalent tool; everything hardware, nicely legible and easy to read, available in one panel. Bits in pink added by me as "Nope, not in Qubes!" quick-notations, as well as redactions made. Obvs some of the things on my Mac wd be Qube specific and handled by Policies, in QubesOS.

**Where is the value to a user, and who might that user be?**
This information is relevant to me to learn about monitor compatibility, RAM and hard drive upgrades, battery replacement, as well as any additional hardware compatibility interests.

**Describe alternatives you've considered**
Bash, which is too much work and frustrating to always have to depend upon.

**Additional context**
This will become a priority to address when considering things necessary to facilitate ease of QubesOS user for non-technical users. Likewise, it will make the life of fleet managers much easier, for enterprise use, if regular users can easily access logs and part numbers, without having to use Terminal.

![macinfo-02](https://user-images.githubusercontent.com/8262612/103389731-8678ab80-4ac5-11eb-9cfc-1482f8360bfd.png)
<img width="795" alt="memory" src="https://user-images.githubusercontent.com/8262612/103389639-e458c380-4ac4-11eb-8e56-3245f6adcb6f.png">
<img width="795" alt="battery" src="https://user-images.githubusercontent.com/8262612/103389632-db67f200-4ac4-11eb-9246-5d24ea5a5bd0.png">
<img width="795" alt="logs" src="https://user-images.githubusercontent.com/8262612/103389635-e15dd300-4ac4-11eb-82de-96ab46286423.png">

**Comment by ninavizz** [comment]
Note: Per #5656 this might work well in a new widget in the Tray, that could also handle managing Updates. TBD.

**Comment by andrewdavidwong** [comment]
I would think that this sort of tool (or at least one that contains most of the desired functionality) would come from the upstream distro that dom0 is using. Seems like a pretty standard OS thing. I'm surprised Fedora doesn't have one by default. Perhaps there's one that can be installed (if deemed secure).

Of course, there may also be Qubes-specific things you want included, which a standard upstream tool wouldn't have.

**Comment by donob4n** [comment]
A simple solution could be a button in Qubes Manager like "Hardware Info" or "System Info" which shows the output of `qubes-hlc-report`. It could include information or a simple way for uploading it to https://www.qubes-os.org/hcl/.

**Comment by ninavizz** [comment]
@donob4n Yeah, I looked at the Qubes HCL output a while ago, and it's great. I'd like for this to be an item discoverable from the "Settings" dropdown menu... which I'm working into my redesign of the App Menu experience, overall. 

This is a mockup I threw together based on the HCL output and what feels most urgent to Qubes users (as I'm aware):
<img width="498" alt="Screen Shot 2021-03-14 at 7 30 58 PM" src="https://user-images.githubusercontent.com/8262612/111096081-fa8af300-84fb-11eb-9f20-69f45c46abe0.png">

**Comment by ninavizz** [comment]
Posting an updated version of what I put together for user testing. Using my own experience as a user to inform the design (which is BAD for researchers to do!), I wanted to include info from both qubes-hcl-report, and the CLI RAM thingydoo. 

In terms of "Grade Of Impact" I'd assign this, I'd say the value in this will be to simply spare users fussing with Google and Terminal to figure something out in 10 minutes, that they could instead quickly see in a single pane view. The bios and Qubes version stuff are currently buried in Qube Manager, whereas I had to google a bunch to learn about the hcl and RAM stuff.

![This Device](https://user-images.githubusercontent.com/8262612/113236714-43f87380-925a-11eb-9fa4-84518ba1fc11.png)

**Comment by deeplow** [comment]
Instead of the `qubes-hcl-report` command it could be more useful to have a button:

<kbd>Copy HCL report to clipboard</kbd>

This could internally run the command and just copy it to the global clipboard. This way the user would only need to `ctrl-shif-v` and then `ctrl-v` on a website. But since this may potentially reveal information that could be used in an attack it could also make sense to make this thing less accessible (leaving as is).

**Comment by ninavizz** [comment]
That is a useful suggestion, thank you @deeplow. :)

I actually wanted to provide a friendly way to expose users to the dom0 Terminal, to get folks more comfortable with it. If that makes sense. This felt like a nice opportunity to offer high-level, critical information folks need for in-the-moment troubleshooting or hardware expansion considerations, which then has that entry-path introduced to get more information if needed.

**Comment by marmarta** [comment]
It's been done a while in the global config tool
